src/main/docker/Dockerfile

```diff
@@ -19,7 +19,7 @@ USER gerrit
 RUN java -jar /var/gerrit/bin/gerrit.war init --batch -d /var/gerrit
 RUN java -jar /var/gerrit/bin/gerrit.war reindex -d /var/gerrit
 
-ADD ${URL} /var/gerrit/plugins/eiffel-plugin.jar
+COPY ${PLUGIN_LOCATION} /var/gerrit/plugins/eiffel-plugin.jar
```

Comment:
How can we provide an URL address to artifactory where eiffel-plugin.jar is located when it is changed to COPY method?
I guess you can change to CMD  and using "curl" or "wget" or something instead, if you don't prefer ADD.

Reply:
Yes a person will have to download the item first. `ADD` brings too many issues and vulnerabilities as described

Reply:
I agree with @m-linner-ericsson in this case. The user have to download the item first.

Reply:
Ok,, it is just more manual step when building and releasing the Docker images, especially when I going through all Eiffel components and build all latest released versions from artifactories.
If we agree on this way, then I am OK with the change.
I can mention in docker-compose files we have in several Eiffel components,uses some functionality that has been deprecated and removed in later version of docker-compose, e.g. the "health-check" function.